src/status_im/wallet/choose_recipient/core.cljs

```diff
@@ -68,18 +65,66 @@
             ethereum/default-transaction-gas))
 
 (re-frame/reg-fx
- :resolve-address
+ ::resolve-address
  (fn [{:keys [registry ens-name cb]}]
    (ens/get-addr registry ens-name cb)))
 
+(re-frame/reg-fx
+ ::resolve-addresses
+ (fn [{:keys [registry ens-names callback]}]
+   ;; resolve all addresses then callt
```

Comment:
```suggestion
   ;; resolve all addresses then call the callback function with the array of addresses as parameter
```

Reply:
I'll have to fix it tonight.  On mobile right now so wouldn't be able to rebase/squash if I accept the commit now.

Reply:
Fixed in latest commit.